bin/plotting/pycbc_plot_hist

```diff
@@ -46,10 +46,31 @@ pycbc.init_logging(args.verbose)
 f = h5py.File(args.trigger_file, 'r')
 ifo = tuple(f.keys())[0]
 
+# This is fixed better on master, but should work here
+if args.x_var == 'snr' or args.x_var == 'newsnr_sgveto':
+    with pycbc.io.HFile(args.trigger_file, 'r') as trig_file:
+        n_triggers_orig = trig_file[f'{ifo}/snr'].size
+        logging.info("Trigger file has %d triggers", n_triggers_orig)
+        logging.info('Generating trigger mask')
+        # psd_var_val may not have been calculated
```

Comment:
as psd_var_val is likely to be used here, can we add something like [this](https://github.com/gwastro/pycbc/blob/v2.3.6/bin/all_sky_search/pycbc_bin_trigger_rates_dq#L57)?

Reply:
I think this is already done on the main branch. We don't need it on the v23 release branch based on current config files (if I missed something let me know!)

Reply:
yup - current configs mean it isn't used, so its not a problem